src/pytest_rabbitmq/factories/process.py

```diff
@@ -96,7 +96,7 @@ def rabbitmq_proc_fixture(request):
         rabbit_port = get_port(port) or get_port(config["port"])
 
         rabbit_path = os.path.join(gettempdir(), f"rabbitmq.{rabbit_port}/")
-        rabbit_plugin_path = plugindir or config["plugindir"] or rabbit_path
+        rabbit_plugin_path = plugindir or config["plugindir"] or os.path.join(rabbit_path, "plugins")
```

Comment:
changed the default to None, and pointed to plugins inside the created rabbit_path as created above. However I believe it should no have default